src/style.css

```diff
@@ -1,3 +1,63 @@
+:root {
+  --primary-blue: #00acdc;
+  --font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+  font-family: var(--font-family);
```

Comment:
The task requires styling using classes rather than tag names. Consider using a class for the body styling instead of the tag selector.